cmd/multiFaReduce/multiFaReduce.go

```diff
@@ -6,14 +6,35 @@ package main
 import (
 	"flag"
 	"fmt"
-	"log"
-
+	"github.com/vertgenlab/gonomics/bed"
 	"github.com/vertgenlab/gonomics/fasta"
+	"log"
 )
 
-func mfaReduce(inFilename, outFilename string) {
+func mfaReduce(inFilename, outFilename, bedFilename, chrom string, refStart int) {
```

Comment:
Note to self: this PR provides an option to output a bed of variable locations in addition to a reduced multiFa